templates/less/pages/signup.less

```diff
@@ -0,0 +1,111 @@
+.signup-page {
+
+  .navbar {
+    background: transparent;
+  }
+
+  @white: /* not #F9F7FF...? */ white;
```

Comment:
we should, but as a followup: there are a LOT of values in our less files that should be variables but aren't.